airbyte-integrations/connectors/source-sendgrid/source_sendgrid/spec.json

```diff
@@ -7,14 +7,14 @@
     "required": ["apikey"],
     "additionalProperties": true,
     "properties": {
-      "apikey": {
```

Comment:
If we already have specs inlined in the manifest, where is `spec.json` used on it's own separately?